test/metabase/domain_entities/core_test.clj

```diff
@@ -0,0 +1,46 @@
+(ns metabase.domain-entities.core-test
+  (:require [expectations :refer [expect]]
+            [metabase.domain-entities.core :as de]
+            [metabase.models
+             [field :refer [Field]]
+             [table :as table :refer [Table]]]
+            [metabase.test
+             [data :as data]
+             [domain-entities :refer :all]]))
+
+
+(expect
+  [:field-id (data/id :venues :price)]
+  (#'de/mbql-reference (Field (data/id :venues :price))))
+
+(expect
+  [:field-literal "PRICE" :type/Integer]
+  (#'de/mbql-reference (dissoc (Field (data/id :venues :price)) :id)))
+
+
+(defn- hydrated-table
+  [table-name]
+  (let [table (-> table-name data/id Table)]
+    (assoc table :fields (table/fields table))))
+
+(expect
+  (de/satisfies-requierments? (hydrated-table :venues) (test-domain-entity-specs "Venues")))
+
+
+(expect
+  "Venues"
+  (-> test-domain-entity-specs vals (#'de/best-match) :name))
+
+
+(expect
```

Comment:
what are these tests testing?

Reply:
Can you please add a comment explaining what we're testing here? That way if the test fails we will have an easier time figuring out what's going on. Or if we need to add new tests it will be easier to tell what's already adequately covered.

Reply:
stale